<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>update_groupe</class>
 <widget class="QDialog" name="update_groupe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>241</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="client_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>287</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Gerer les Client </string>
    </attribute>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter client</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifer Client</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer Client</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Afficher Client</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="groupedevoyageurs_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>287</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Gerer les Groupe  des Voyageurs</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter Groupe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifer Groupe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer Groupe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_16">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Afficher Groupe</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id_p">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>270</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_type">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_17">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ANULLER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string> ID De personnel</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>381</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nbr">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>190</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_18">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>380</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>confirmer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>12</y>
     <width>771</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Desktop/téléchargement (2).jpg</pixmap>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>textBrowser</zorder>
  <zorder>toolBox</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEdit_id_p</zorder>
  <zorder>lineEdit_type</zorder>
  <zorder>pushButton_17</zorder>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEdit_nbr</zorder>
  <zorder>pushButton_18</zorder>
  <zorder>lineEdit_id</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_17</sender>
   <signal>clicked()</signal>
   <receiver>update_groupe</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>614</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>1144</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
